operator/roles/kiali-deploy/defaults/main.yml

```diff
@@ -23,13 +23,26 @@ kiali_defaults:
 
   auth:
     strategy: ""
+    ldap:
+      ldap_host: ""
+      ldap_port: 0
+      ldap_use_ssl: false
+      ldap_insecure_skip_verify: false
+      ldap_user_filter: (cn=%s)
+      ldap_group_filter: (cn=%s)
+      ldap_base: ""
+      ldap_bind_dn: ""
+      ldap_search_filter: (&(name={USERID}))
+      ldap_role_filter: ""
+      ldap_mail_id_key: mail
+      ldap_user_id_key: cn
+      ldap_member_of_key: memberOf
```

Comment:
all settings are sorted alphabetically in this file, and make sure to wrap string values in double-quotes. So make this `auth` section look like this:

```
  auth:
    ldap:
      ldap_base: ""
      ldap_bind_dn: ""
      ldap_group_filter: "(cn=%s)"
      ldap_host: ""
      ldap_insecure_skip_verify: false
      ldap_mail_id_key: "mail"
      ldap_member_of_key: "memberOf"
      ldap_port: 0
      ldap_role_filter: ""
      ldap_search_filter: "(&(name={USERID}))"
      ldap_use_ssl: false
      ldap_user_filter: "(cn=%s)"
      ldap_user_id_key: "cn"
    strategy: ""
```

Reply:
Sorted